editor/src/clj/editor/atlas.clj

```diff
@@ -676,8 +662,20 @@
         complete-ddf-animations (map complete-ddf-animation
                                      incomplete-ddf-animations
                                      animations-in-ddf)
+        fixed-image-name-hashes (-> []
+                                    (into
+                                      (map #(-> % :path (texture-set-gen/resource-id rename-patterns) murmur/hash64))
+                                      all-atlas-images)
+                                    (into
+                                      (mapcat
+                                        (fn [{:keys [id images]}]
+                                          (eduction
+                                            (map #(-> % :path (texture-set-gen/resource-id id rename-patterns) murmur/hash64))
+                                            images)))
+                                      animations))
         complete-ddf-texture-set (assoc incomplete-ddf-texture-set
-                                   :animations complete-ddf-animations)]
+                                   :animations complete-ddf-animations
+                                   :image-name-hashes fixed-image-name-hashes)]
```

Comment:
We need to update `:image-name-hashes` because Bob's atlas layout calculation fills the hashes with full image paths, but it is expected that it will fill them with frame IDs that might be derived from animations, e.g. `/foo/1.png` in animation `anim` has frame ID `anim/1`. Meanwhile, we don't provide animation names to bob at all for performance.

Reply:
> Is the intention here to add the hash twice in that situation?
Yes, this is 100% the intention here. This is how Bob works. The image (i.e. rect) is added to the layout only once, but it might be referenced in multiple ways — by a single `base-name`, or by `anim/base-name` for every animation that uses the image. I'll add a code comment.